src/components/Footer.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+function Footer() {
+  return (
+    <footer className="footer">
+      <div>
+        <h3>Thanks for visiting.</h3>
+        <p>Built with React.js using icanhazdadjoke API © {new Date().getFullYear()}</p>
```

Comment:
So this is kind of an advanced technique but... the problem with `Footer` getting the full year itself is that it makes testing it a hassle. One of the programming models around react is "given the exact same props, you'll get the same rendered output". The way `Footer` is written, that is no longer true. It renders differently in 2019 compared to 2020 which very recently just happened. 

TL;DR, pass in `now` or `year` as a prop. Then you'll have control from OUTSIDE the component if/when you end up writing a test.